waiter/src/waiter/service_description.clj

```diff
@@ -478,6 +510,10 @@
                                              parameter->issues :name "name must be a non-empty string.")
                                            (attach-error-message-for-parameter
                                              parameter->issues :ports "ports must be an integer in the range [1, 10].")
+                                           (attach-error-message-for-parameter
+                                             parameter->issues :profile
+                                             (str "profile must be a non-empty string"
```

Comment:
Must it?

Reply:
If a user provides a profile, it must be non-empty. If they want it empty, then don't specify it.